web/models/user.ex

```diff
@@ -32,6 +39,14 @@ defmodule CodeCorps.User do
     |> put_pass_hash()
   end
 
+  def update_changeset(struct, params) do
+    struct
+    |> changeset(params)
+    |> cast(params, [:first_name, :last_name, :twitter, :biography, :website])
+    |> validate_format(:website, ~r/\A((http|https):\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,}(([0-9]{1,5})?\/.*)?#=\z/ix)
+    |> validate_format(:twitter, ~r/\A[a-zA-Z0-9_]{1,15}\z/)
+  end
```

Comment:
`validate_format` passes if the string is blank or the parameter is not in the map. 

Additionally
* If the parameter is a blank string, it will update the model with that blank string
* If the parameter is missing from the map, it will not update the model at all

To make a parameter required, we would use `validate_required` instead.

I think the current behavior should be ok. If the user makes no changes to their twitter or website, nothing will happen to it on update. If they explicitly clear the textbox, it will get updated to a blank string. I think it should be ok for the user to allow unsetting their profile info.

I can write a validation, but I don't think it's necessary, since this is documented, default behavior of the framework.